### DISCUSSION TITLE
Card pagina web

### DISCUSSION THREAD

**Discussion by Matteogolf** [indomus/forum#1685] [HUB: Home Assistant]
ciao, Utilizzando la card pagina web come fisso la dimensione del riquadro e rimpicciolisco l'immagine interna per evitare che compaiano le bare di scorrimento?

type: iframe
url: >-
  https://www.firenzemeteo.it/webcam/appennino/toscana/img/webcam-val-di-bisenzio-poggio-di-petto.jpg
aspect_ratio: 75%
title: webcam

grazie

**Comment by 50494554524F** [comment]
per layout più complessi puoi usare

https://github.com/thomasloven/lovelace-layout-card
   +
https://github.com/thomasloven/lovelace-card-mod

**Reply by Matteogolf** [reply]
grazie per i link ma non trovo quello che ridimensiona dentro la card la dimensione dell'immagine.
Vedo come ridimensionare le card ma non i contenuti delle card.
Mi è sfuggito?

grazie ciao

**Reply by 50494554524F** [reply]
cerca le guide per applicare css agli iframe ed ai contenuti... oppure, se non viene bloccato il linking dal sito che vuoi usare, creati una pagina pagina html con l'immagine linkata, la ridimensioni e la includi

**Reply by 50494554524F** [reply]
ho provato, non lo blocca... creati un file .html con giusto questa riga e ridimensiona a piacere, poi linkalo con la card

`<img src="https://www.firenzemeteo.it/webcam/appennino/toscana/img/webcam-val-di-bisenzio-poggio-di-petto.jpg" width="250px" />`
